src/packages/frontend/components/icon.tsx

```diff
@@ -681,14 +681,15 @@ try {
 }
 
 // This was nice but unfortunately it exceeds typescript limits.
-//export type IconName = keyof typeof IconSpec;
```

Comment:
The limits are no longer exceeded...?  Or using "as const" fixes it?

Reply:
I think either were are problem, but apparently it is fine now. There was a bit of fallout due to typing "icon" everywhere, but that's easy to fix.

Reply:
Nice!!